packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -78,6 +78,19 @@ const Duration _kCursorBlinkHalfPeriod = Duration(milliseconds: 500);
 // is shown in an obscured text field.
 const int _kObscureShowLatestCharCursorTicks = 3;
 
+/// The default mime types to be used when contentInsertionMimeTypes is not provided.
+///
+/// The default value supports inserting images of any supported format.
+const List<String> kDefaultContentInsertionMimeTypes = <String>[
```

Comment:
Does this have to be public? I think with `ContentInsertionConfiguration` now you can just add the list to the API docs and make this private?

Reply:
Yes, we can make it private

Reply:
Actually no I can't - I need to be able to set it inside the default values of the properties, otherwise there's a bunch of failing tests